client/src/main/java/io/pravega/client/stream/impl/ControllerImpl.java

```diff
@@ -810,6 +807,11 @@ private TxnSegments convert(CreateTxnResponse response) {
                 });
     }
 
+    @Override
+    public void close() throws Exception {
```

Comment:
FYI, if you don't need to throw an Exception, there's no need to declare it :)

Reply:
done